### DISCUSSION TITLE
battery prompt on Mac OS X

### DISCUSSION THREAD

**Discussion by cxwx** [starship/starship#1501] [Q&A] [answered]
I've set battery on starship.toml on Mac OS X, but nothing displayed.
Does it not support on Mac OS X?

**Comment by matchai** [comment, marked as answer]
The default configuration is to only show the battery if it is <10% charged.
Please take a look at the documentation for the module here: https://starship.rs/config/#battery

**Comment by cxwx** [comment]
my mistake
